hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadataWriter.java

```diff
@@ -89,6 +89,10 @@
 
   private static final Logger LOG = LogManager.getLogger(HoodieBackedTableMetadataWriter.class);
 
+  // Virtual keys support for metadata table
+  private static final String RECORD_KEY_FIELD = "key";
+  private static final boolean POPULATE_META_FIELDS = false;
```

Comment:
this is just for the `FILES` schema or is it true for any metadata partition? probably

Reply:
better to have one config for all partitions of the metadata table (else it gets complicated).

Reply:
Added a populate meta fields config for the metadata table. And yes, this is a common config for all partitions under metadata table, otherwise the design gets complicated.